### PR TITLE
Generate Kaptain 2.2 docs

### PR DESCRIPTION
**Pull request by alembiewski**
Signed-off-by: Alex Lembiyeuski <alembiyeuski@d2iq.com>

### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4660.s3-website-us-west-2.amazonaws.com/

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

**Review by cmastr — APPROVED** [review]

File: pages/dkp/kaptain/2.2.0/sdk/1.3.x/api/index.md [hunk 1]
```diff
@@ -0,0 +1,1200 @@
...
+- `workers`: Number of parallel workers to use for each trial (default: 2).
+- `gpus`: Number of GPUs to use (default: 0).
+- `memory`: Amount of memory for each worker (optional),
+- `cpu`: Number of CPUs to use for each worker (optional).
+- `resources`: Advanced API for resource specification. Do not use in tandem with the parameters gpus, memory and cpu (optional).
+- `trials`: Maximum number of trials (default: 16).
+- `parallel_trials`: Maximum number of trials to run in parallel (default: 2).
+- `failed_trials`: Maximum number of failed trials before hyperparameter tuning stops (default: 4).
+- `algorithm`: Algorithm to use for hyperparameter search (default: random).
+- `algorithm_setting`: Algorithm settings. Please see https://www.kubeflow.org/docs/components/hyperparameter-tuning/experiment/ for details.
```
Comment (gvasquezvargas) [inline pages/dkp/kaptain/2.2.0/sdk/1.3.x/api/index.md:707]: Hey @alembiewski! This link is giving me an 404 error, we should probably replace with: 
```suggestion
- `algorithm_setting`: Algorithm settings. Please see https://www.kubeflow.org/docs/components/katib/experiment/ for details.
```

File: pages/dkp/kaptain/2.2.0/sdk/1.3.x/api/index.md [hunk 2]
```diff
@@ -0,0 +1,1200 @@
...
+ | deploy(model_uri: Optional[str] = None, autoscale: int = 2, gpus: Optional[int] = None, cpu: Optional[str] = None, memory: Optional[str] = None, resources: Optional[Resources] = None, replace: bool = False, transformer: Optional[Transformer] = None, **kwargs: str, ,) -> bool
+```
+
+Deploys a model.
+
+##### Simple / advanced resource API
+
+Resources may be specified via the 'simple' resource parameters::
+
+model.deploy(model_uri=uri, cpu=1, memory="2G", gpus=0)
```
Comment (gvasquezvargas) [inline pages/dkp/kaptain/2.2.0/sdk/1.3.x/api/index.md:730]:
```suggestion
    model.deploy(model_uri=uri, cpu=1, memory="2G", gpus=0)
```

File: pages/dkp/kaptain/2.2.0/sdk/1.3.x/api/index.md [hunk 3]
```diff
@@ -0,0 +1,1200 @@
...
+
+... the deployed model process will have both the request and limit set for all resource parameters.
+
+More fine-grained resource specification is possible via the 'resources' parameter::
+
+model.deploy(model_uri=uri, resources=Resources(cpu_request=1, memory_limit="2G", gpu_limit=gpus))
```
Comment (gvasquezvargas) [inline pages/dkp/kaptain/2.2.0/sdk/1.3.x/api/index.md:736]:
```suggestion
    model.deploy(model_uri=uri, resources=Resources(cpu_request=1, memory_limit="2G", gpu_limit=gpus))
```

File: pages/dkp/kaptain/2.2.0/sdk/1.3.x/api/index.md [hunk 4]
```diff
@@ -0,0 +1,1200 @@
...
+A canary deployment allows a model to be run in parallel with a baseline or previous model
+revision. This allows traffic to be split, so the latest revision can be checked for
+possible issues with model (e.g. compared to the baseline) or system (e.g. latency)
+performance.
+To deploy a model to the canary, a previously deployed model revision must exist.
+
+------------------------------
+To deploy canary with 30 percent traffic:
+
+model.deploy_canary(canary_traffic_percentage=30)
```
Comment (gvasquezvargas) [inline pages/dkp/kaptain/2.2.0/sdk/1.3.x/api/index.md:774]: @alembiewski  wondering if these should be code snippets as well? Or are these plain text as shown in the preview?

Comment (alembiewski) [reply pages/dkp/kaptain/2.2.0/sdk/1.3.x/api/index.md:774]: these pages are auto-generated by `pydoc-markdown` from python code. Given that this is an API reference, I would say it's fine to keep it as is

**Review by gvasquezvargas — APPROVED** [review]

**Pull request merged**